trio/_tests/test_path.py

```diff
@@ -53,7 +53,7 @@ async def test_magic() -> None:
 async def test_cmp_magic(cls_a: EitherPathType, cls_b: EitherPathType) -> None:
     a, b = cls_a(""), cls_b("")
     assert a == b
-    assert not a != b
+    assert a == b
```

Comment:
This is testing `!=` which is its own method on `a`

Reply:
Fixed as of 76d7413